### ISSUE TITLE
Not assuming not equal to

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#1977]
Imported from SourceForge on 2024-07-05 15:42:00
Created by **[paulus2005](https://sourceforge.net/u/paulus2005/)** on 2007-11-07 12:15:09
Original: https://sourceforge.net/p/maxima/bugs/1289

---

See the following code:

\(%i1\) assume\(k \# 1\);

\`assume': \`\#' means syntactic nonequality in Maxima. Maybe you want to
use \`not equal'.
\-- an error.  To debug this try debugmode\(true\);
\(%i2\) assume\(notequal\(k,1\)\);
\(%o2\)                          \[notequal\(k, 1\)\]
\(%i3\) is\(equal\(k,1\)\);
\(%o3\)                               unknown

In short, maxima cannot assuming not equal to.

Paul

\-------------------------------------------------------------Maxima version: 5.13.0Maxima build date: 15:18 8/30/2007host type: i386-redhat-linux-gnulisp-implementation-type: GNU Common Lisp \(GCL\)lisp-implementation-version: GCL 2.6.7-------------------------------------------------------------

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 15:42:01
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2007-11-08 02:47:11
Original: https://sourceforge.net/p/maxima/bugs/1289/#62c7

---

Logged In: YES 
user\_id=895922
Originator: NO

I have a fix for--I'll apply after I do some more testing.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 15:42:04
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2007-11-08 02:47:11
Original: https://sourceforge.net/p/maxima/bugs/1289/#64b5

---

- **assigned_to**: nobody --> willisbl

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 15:42:08
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2007-11-16 20:36:17
Original: https://sourceforge.net/p/maxima/bugs/1289/#7fd3

---

Logged In: YES 
user\_id=895922
Originator: NO

Fixed by compar.lisp CVS version 1.31

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-05 15:42:11
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2007-11-16 20:36:18
Original: https://sourceforge.net/p/maxima/bugs/1289/#80d0

---

- **status**: open --> closed
